students/Chao/session09/mailroom_oo.py

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env python
+
+class Donor:
+    def __init__(self, name, amount):
+        self.name = name
+        self.amount = [amount]
+
+    def new_donation(self, amount):
+        self.amount.append(amount)
+
+    @property
+    def total_donation(self):
+        return sum(self.amount)
+
+
```

Comment:
Good start! Now you need a class to store and manipulate all of the donor objects.